src/metakb/cli.py

```diff
@@ -14,178 +14,414 @@
+@click.argument(
+    "normalizer",
+    type=click.Choice(list(NormalizerName), case_sensitive=False),
+    nargs=-1,
+)
+def update_normalizers(
+    db_url: str | None, normalizer: tuple[NormalizerName, ...]
+) -> None:
+    """Reload gene, disease, and therapy normalizer data.
+
+    Forces delete of each prior to fetching and loading new data. If errors are
+    encountered, attempts to complete updates of other normalizers before exiting.
+
+    Providing no arguments will attempt to update all three:
+
+        $ metakb update-normalizers
+
+    Providing individual normalizer names as arguments will update only those
+    normalizers:
+
+        $ metakb update-normalizers disease therapy
+
+    \f
+    :param db_url: URL endpoint of normalizers DynamoDB database. If not given, the
+        individual normalizers will revert to their own defaults.
+    :param normalizer: tuple (possibly empty) of normalizer names to update
+    """  # noqa: D301
+    success = True
+    update_params = [
+        (NormalizerName.DISEASE, DISEASE_AWS_ENV_VAR_NAME),
+        (NormalizerName.THERAPY, THERAPY_AWS_ENV_VAR_NAME),
+        (NormalizerName.GENE, GENE_AWS_ENV_VAR_NAME),
+    ]
+    if normalizer:
+        update_params = [p for p in update_params if p[0] in normalizer]
+    for name, aws_env_var_name in update_params:
+        if aws_env_var_name in environ:
+            msg = (
+                f"Updating the {name.value} AWS database from the MetaKB CLI is "
+                f"prohibited. Unset the environment variable:`{aws_env_var_name}` to "
+                "proceed."
+            )
+            _logger.error(msg)
+            click.echo(msg)
+            success = False
+            continue
+
+        _echo_info(f"Loading {name.value} normalizer data...")
+        try:
+            update_normalizer(name, db_url)
+        except (Exception, SystemExit) as e:
+            _logger.error(
+                "Encountered error while updating %s database: %s", name.value, e
+            )
+            click.echo(f"Failed to update {name.value} normalizer.")
+            success = False
+            continue
+        _echo_info(f"Successfully loaded {name.value} normalizer data.\n")
+
+    if success:
+        _echo_info("Normalizer databases updated.\n")
+    else:
+        click.echo("Not all updates were successful. See logs for more details.")
+        click.get_current_context().exit(1)
+
+
+@cli.command()
 @click.option(
-    "--db_username",
-    default="neo4j",
+    "--refresh_source_caches",
+    "-r",
+    is_flag=True,
+    default=False,
     help=(
-        "Username to provide to application Neo4j database. Can also be provided via environment variable METAKB_DB_USERNAME, which takes priority."
+        "True if source caches (e.g. CivicPy) should be updated prior to data regeneration. Note this will take several minutes. False if local cache should be used"
```

Comment:
```suggestion
        "True if source caches (e.g. CIViCPy) should be updated prior to data regeneration. Note this will take several minutes. False if local cache should be used"
```